docs/linux/setup.md

```diff
@@ -7,6 +7,7 @@ Instructions for a particular VM type or kernel arch can be found on these pages
 - [Setup: Ubuntu host, Odroid C2 board, arm64 kernel](setup_ubuntu-host_odroid-c2-board_arm64-kernel.md)
 - [Setup: Linux host, QEMU vm, arm64 kernel](setup_linux-host_qemu-vm_arm64-kernel.md)
 - [Setup: Linux host, Android device, arm64 kernel](setup_linux-host_android-device_arm64-kernel.md)
+- [Setup: Ubuntu host, Android device, arm32 kernel](setup_ubuntu-host_android-arm32.md)
```

Comment:
Please name this `setup_ubuntu-host_android-device_arm32-kernel.md` to be consistent.